### PR TITLE
attrauthcomanage_noty_dictionary

### PR DESCRIPTION
**Pull request by ioigoume**
Add noty.definition.json

### CODE REVIEW

File: dictionaries/noty.definition.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "yes": {
+        "en": "Resend Verfication Email"
```
Comment (nikosev) [inline dictionaries/noty.definition.json:3]: Verification

File: dictionaries/noty.definition.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        "en": "Account in Pending Email Confirmation Status"
+    },
+    "invitation_restored": {
+        "en": "Invitation has been restored."
+    },
+    "petition_info": {
+        "en": "The identifier <cite><b>%ORGIDENT%</b></cite> is in Email Pending Confirmation Status.&nbspPlease check your Mail <cite><b>%MAIL%</b></cite> Inbox or Spam for further information.&nbsp"
+    },
+    "petition_user_notify": {
+        "en": "<p class=\"petition-user-notify\">We have sent an email with a confirmation link to your email address <cite><b>%MAIL%</b></cite>. In order to complete the sign-up process, please click the confirmation link. If you do not find the email in your Inbox, please check your Spam or Junk folder for an email from \"EGI Check-in Notifications\". If you do find the email in these folders, mark the email as \"safe\" or \"not spam\" to ensure that you receive any future notifications about your EGI Check-in ID.</p><p>You can also click the <strong>\"Resend Verfication Email\"</strong> button below to have another email sent to you."
```
Comment (nikosev) [inline dictionaries/noty.definition.json:27]: Verification

File: dictionaries/noty.definition.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        "en": "Action Required"
+    },
+    "account_pending_confirmation": {
+        "en": "Account in Pending Email Confirmation Status"
+    },
+    "invitation_restored": {
+        "en": "Invitation has been restored."
+    },
+    "petition_info": {
+        "en": "The identifier <cite><b>%ORGIDENT%</b></cite> is in Email Pending Confirmation Status.&nbspPlease check your Mail <cite><b>%MAIL%</b></cite> Inbox or Spam for further information.&nbsp"
```
Comment (nikosev) [inline dictionaries/noty.definition.json:24]: `&nbsp` --> `&nbsp;`

File: dictionaries/noty.definition.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    "petition_user_notify": {
+        "en": "<p class=\"petition-user-notify\">We have sent an email with a confirmation link to your email address <cite><b>%MAIL%</b></cite>. In order to complete the sign-up process, please click the confirmation link. If you do not find the email in your Inbox, please check your Spam or Junk folder for an email from \"EGI Check-in Notifications\". If you do find the email in these folders, mark the email as \"safe\" or \"not spam\" to ensure that you receive any future notifications about your EGI Check-in ID.</p><p>You can also click the <strong>\"Resend Verfication Email\"</strong> button below to have another email sent to you."
+    },
+    "petition_token_expired": {
+        "en": "<div class=\"invite-expire\">Email confirmation token EXPIRED <b>&nbsp%DATE%</b>&nbspago</div>"
```
Comment (nikosev) [inline dictionaries/noty.definition.json:30]: `&nbsp` --> `&nbsp;`

File: dictionaries/noty.definition.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    "petition_token_expires": {
+        "en": "<div class=\"invite-expire\">Email confirmation token EXPIRES in <b>&nbsp%DATE%</b></div>"
```
Comment (nikosev) [inline dictionaries/noty.definition.json:33]: `&nbsp` --> `&nbsp;`

**Review by nikosev — CHANGES_REQUESTED** [review]
Please take a look at the inline comments

File: dictionaries/noty.definition.json (hunk 1 above)
Comment (ioigoume) [reply dictionaries/noty.definition.json:3]: Done

File: dictionaries/noty.definition.json (hunk 3 above)
Comment (ioigoume) [reply dictionaries/noty.definition.json:24]: done

File: dictionaries/noty.definition.json (hunk 5 above)
Comment (ioigoume) [reply dictionaries/noty.definition.json:33]: done

File: dictionaries/noty.definition.json (hunk 4 above)
Comment (ioigoume) [reply dictionaries/noty.definition.json:30]: done

File: dictionaries/noty.definition.json (hunk 2 above)
Comment (ioigoume) [reply dictionaries/noty.definition.json]: Done

**Review by nikosev — APPROVED** [review]
LGTM

**Pull request merged**